### DISCUSSION TITLE
personeel

### DISCUSSION THREAD

**Discussion by Sascha-Davidson** [fdnd/docs.fdnd.nl#86] [Reflectie]
De docenten van FDND hebben veel kennis van front-end development.  De opdrachten die wij maken binnen de opleiding worden goed bijgehouden door de docenten. Voor alle vragen en problemen binnen front-end development kunnen wij terecht bij de docenten. De website die wij  maken zijn  voor echte bedrijven, hierdoor leren wij ook goed te communiceren met opdrachtgevers.  De samen werking tussen de student, docent en opdrachtgever zorgt er voor dat er veel structuur en leer gelegenheden zijn. 

De docenten kommen van een andere opleiding waar ze ook meer doen met design en concepten, hierdoor krijgen wij ook hierin nog wat kennis die wij gebruiken bij projecten. We hebben af en toe een gast spreker, hij/zij praat over een techniek, ervaring, design enz. Hierin krijgen wij veel informatie over projecten technieken van front-end development.

De toetsen zijn goed onderhouden. De criteria die wij moeten leren is goed en duidelijk verdeeld in sub criteria. Via deze sub criteria kunnen wij goed aantonen wat wij kunnen en waar we beter in moet worden.

De docenten blijven goed op de hoogte van internationale ontwikkelingen binnen het vakgebied. Onze  docenten stimuleren ons om op allemaal verschillende methodes op de hoogde van de web te blijven. Dit doen ze door goede blogs, website, events enz. Het is goed dat ze dit ons aan leren zodat wij ook goed op de hoogte blijven van de ontwikkelingen in het vakgebied.

**Comment by ju5tu5** [comment]
Opgenomen in de lopende tekst.
